### ISSUE TITLE
Variable summing update to work in all cases

### ISSUE THREAD

**Issue by rschwant** [NOAA-CSL/MELODIES-MONET#319]
You need to specify variables in order for summing to work. This is not a huge problem because you can just add the variables in the yaml file and all works well. But let's update this so that if you do not provide any changes to the variables, you can still sum variables. I believe all that is needed is to add a line in the driver

 if variable_dict = None: 
variable_dict = []. 

But we could find other solutions too. This is non-urgent and can be fixed after AGU and holidays but noting it here so we do not forget to update.

**Comment by rschwant** [comment]
Part of this is done here: https://github.com/NOAA-CSL/MELODIES-MONET/pull/353

**Comment by zmoon** [comment]
I think we have both in now?

**Comment by rschwant** [comment]
Yes, we do, this is now complete with this PR: https://github.com/NOAA-CSL/MELODIES-MONET/pull/347
